packages/ember/tests/acceptance/sentry-performance-test.js

```diff
@@ -122,10 +122,10 @@ module('Acceptance | Sentry Transactions', function (hooks) {
         'ui.ember.runloop.render | undefined',
         'ui.ember.runloop.afterRender | undefined',
         'ui.ember.runloop.destroy | undefined',
-        'ember.route.afterModel | slow-loading-route.index',
+        'ui.ember.route.after_model | slow-loading-route.index',
         'ui.ember.runloop.actions | undefined',
-        'ember.route.setupController | slow-loading-route',
-        'ember.route.setupController | slow-loading-route.index',
+        'ui.ember.route.setup_controller | slow-loading-route',
+        'ui.ember.route.setup_controller | slow-loading-route.index',
         'ui.ember.runloop.routerTransitions | undefined',
         'ui.ember.component.render | component:link-to',
         'ui.ember.component.render | component:link-to',
```

Comment:
Should I change `ember.route.model` operation in this file?

Reply:
Yes we should change that to prepend `ui`